### PR TITLE
Feat/test parse resource

### PR DESCRIPTION
**Pull request by ljc1991**
# Description

Try to parse resource page. 

## minor

- page parser testing.

# Impaction

## Screenshots

| Scenarios  | Before | After |
| ---------- | ------ | ----- |
| Parse resource page | N | 截圖 2023-09-08 11 01 34 |

## Performance

N/A

### CODE REVIEW

**Comment by ljc1991** [comment]
### feat: introduce react-markdown and rehype-raw for parsing markdown and HTML

1. Introduce `react-markdown`
2. Refactor component to use  `react-markdown`
3. Parse `markdown` and `HTML` successfully but notice that font-size is changed from 16px to 14px

| Before | After  |
| ------ | ------ |
| 截圖 2023-09-08 17 08 41 | 截圖 2023-09-08 17 09 12 |

File: homepage/src/components/imageAndText.jsx [hunk 1]
```diff
@@ -9,16 +12,16 @@ const ImageAndText = ({ id, image, title, subtitle, content, highlights }) => (
           <div className="image-and-text-details">
             <span className="name">{subtitle}</span>
             <h2 className="sub-position">{title}</h2>
-            <div
-              dangerouslySetInnerHTML={{
-                __html: content,
-              }}
-            />
+            <ReactMarkdown rehypePlugins={rehypeRaw}>{content}</ReactMarkdown>
```
Comment (ben196888) [inline homepage/src/components/imageAndText.jsx:15]: This logic applied everywhere. Please extract this component out to be an reusable component.

Comment (ljc1991) [reply homepage/src/components/imageAndText.jsx]: done

**Comment by ljc1991** [comment]
Create the new branch(#231 ) and merge.

**Pull request closed without merging**